homepage/tests/appUser_model_tests.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from homepage.models import AppUser
+
+
+@pytest.mark.django_db
+class TestAppUser:
+
+    @pytest.fixture
+    def appUsers(self):
+        appUser1 = AppUser.create_appUser('user1', 'user1@mta.ac.il', 'useruser222')
+        appUser2 = AppUser.create_appUser('user2', 'user2@mta.ac.il', '1234')
+        appUser3 = AppUser.create_appUser('user3', 'user3@gmail.com', '10101010')
+
+        return (appUser1, appUser2, appUser3)
+
+    def test_create_appUser(self, appUsers):
+
+        # 1. The 3 appUsers which are created in the fixture above are the correct type of object (AppUser)
+        # 2. The 3 appUsers which are created in the fixture above were sucesfully saved in the DB
+        for appUser in appUsers:
+            assert isinstance(appUser, AppUser)
+            assert AppUser.objects.filter(pk=appUser.pk).exists()
+
+        # 3. An appUser with invalid username is not created
+        try:
+            AppUser.create_appUser('', 'user4@mta.ac.il', 'useruser222')
+        except ValueError:
+            invalid_username_created = False
+
+        assert invalid_username_created is False

```

Comment:
please separate this method into few tests 
each one test separate scenario:
1. test_valid_app_users
2. test_invald_username
3. test_invalid_password

Reply:
changed